cabal-install/src/Distribution/Client/CmdRepl.hs

```diff
@@ -304,7 +304,7 @@ replAction flags@NixStyleFlags{extraFlags = r@ReplFlags{..}, ..} targetStrings g
           lBuildInfo =
             emptyBuildInfo
               { targetBuildDepends = [baseDep]
-              , defaultLanguage = Just Haskell2010
```

Comment:
This is a breaking change...
Does this mean we use `Haskell2010` by default for `repl`?

Reply:
Before submitting the patch, I have launched `repl` with `-v3` and found no mention of `Haskell2010`.

`BuildInfo` is a monoid and that specific field is dealt with with `mplus`, so overwritten. The “it is being overwritten” hypotesis is being corroborated by this fact: if this actually landed `Haskell2010` in the repl, interpreted and compiled behaviour of a program would not be the same and we should have received some bug reports by now.

Does this make sense?

Reply:
That makes sense